src/main/java/com/github/dockerjava/api/model/Network.java

```diff
@@ -76,10 +76,10 @@ public String toString() {
         private String macAddress;
 
         @JsonProperty("IPv4Address")
-        private String ipV4Address;
+        private String ipv4Address;
```

Comment:
need to be verified, if it doesn't exist in master, then rename is ok. If they are already exist in master, then i see no reason for renaming.